_studio/shared/include/mfx_utils.h

```diff
@@ -121,6 +121,11 @@ bool LumaIsNull(const mfxFrameSurface1 * surf)
 #define SAFE_RELEASE(PTR)   { if (PTR) { PTR->Release(); PTR = NULL; } }
 #endif
 
+#ifdef MFX_ENABLE_CPLIB
+    #define IS_PROTECTION_CENC(val) (MFX_PROTECTION_CENC_WV_CLASSIC == (val) || MFX_PROTECTION_CENC_WV_GOOGLE_DASH == (val))
+    #define IS_PROTECTION_ANY(val) IS_PROTECTION_CENC(val)
```

Comment:
You might wish to use MFX_VERSION check here and have #else path to still define IS_* functions. In this way you might reduce number of MFX_ENABLE_CPLIB macro checks.

Reply:
fixed